presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestSetFlatteningOptimizer.java

```diff
@@ -131,6 +130,6 @@ public void assertPlan(String sql, PlanMatchPattern pattern)
                 new PruneUnreferencedOutputs(),
                 new IterativeOptimizer(new StatsRecorder(), ImmutableSet.of(new RemoveRedundantIdentityProjections())),
                 new SetFlatteningOptimizer());
-        assertPlan(sql, LogicalPlanner.Stage.OPTIMIZED, pattern, optimizers);
```

Comment:
i don't understand this change

Reply:
there is overloaded method, which sets this `LogicalPlanner.Stage.OPTIMIZED`. Like default parameter in other languages. Should I extract it to separate commit to make it more clear?